src/app.ts

```diff
@@ -9,15 +9,17 @@ import { packagesRouter } from "./route/packages.route";
 import { resetRouter } from "./route/reset.route";
 import { userRouter } from "./route/user.route";
 import { connectToMongo, disconnectFromMongo } from "./config/config";
-
+var cors = require("cors");
 // define app
-const app = express();
 
 function defineServer() {
+  const app = express();
+  console.log("here");
```

Comment:
Do we need this?

Reply:
Mad respect for actually reviewing the code. I deleted the line